### PR TITLE
added comp_dna.py and its test, reformatted using black and tested using python and pytest

### PR DESCRIPTION
**Pull request by JosephDettmann**
added comp_dna.py and its test, reformatted using black and tested using python and pytest

### CODE REVIEW

File: tests/test_comp_dna.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from src.comp_dna import complementary_dna
+
+
+def test_complementary_dna():
+    assert complementary_dna("AAAA") == "TTTT"
+    assert complementary_dna("TTTT") == "AAAA"
+    assert complementary_dna("CCCC") == "GGGG"
+    assert complementary_dna("GGGG") == "CCCC"
```
Comment (W-Ely) [inline tests/test_comp_dna.py:9]: Should add a test for "ATCG".

File: src/comp_dna.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    comp_dna = ""
+    for char in dna:
+        if char == "C":
+            comp_dna = comp_dna + "G"
+        elif char == "T":
+            comp_dna = comp_dna + "A"
+        elif char == "G":
+            comp_dna = comp_dna + "C"
+        elif char == "A":
+            comp_dna = comp_dna + "T"
```
Comment (W-Ely) [inline src/comp_dna.py]: Your approach is fine.

For education see if understand how these work:
```python
transcription_map = {"A": "T", "C": "G", "G": "C", "T": "A"}
comp_dna = ""
for char in dna:
    comp_dna += transcription_map[char]
```
or
```python
transcription_map = {"A": "T", "C": "G", "G": "C", "T": "A"}
comp_dna = "".join([ transcription_map[char] for char in dna])
```

**Review by W-Ely — CHANGES_REQUESTED** [review]
Add the one mentioned assertion.

File: src/comp_dna.py (hunk 2 above)
Comment (JosephDettmann) [reply src/comp_dna.py:11]: I tried to use this code, It didn't work.

**Review by W-Ely — DISMISSED** [review]

**Pull request merged**